From c6d3256d7535158fb4e93caa032e65a1f3418075 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Wed, 19 May 2021 13:05:09 +0300
Subject: [PATCH] [SW-42609] rma: remove prints in flash read

In case of an active rma multiple prints are seen as each flash_read
read results in 3 prints and each active rma consists of 4 flash_read
calls (1 for boot pointers, 3 for descriptor read).

This leads to a lot of repetitive prints which do not contain any
debug data.

Hence for rma info read from flash these strings has been removed.

Change-Id: I6c6d798fdbe10a96e1b8f62efd7d1223e1b83634
Signed-off-by: Karthikeyan K <karthikk@habana.ai>
---

diff --git a/gaudi/rma_info.c b/gaudi/rma_info.c
index b2104f4..7c08163 100644
--- a/gaudi/rma_info.c
+++ b/gaudi/rma_info.c
@@ -77,10 +77,10 @@
 
 	/* on every read we first choose a valid sector to read from */
 	spi_copy(&header0, flash_mem_context.base[0],
-				sizeof(struct flash_header), "RMA header0");
+				sizeof(struct flash_header), NULL);
 
 	spi_copy(&header1, flash_mem_context.base[1],
-				sizeof(struct flash_header), "RMA header1");
+				sizeof(struct flash_header), NULL);
 
 	if (header0.magic == FLASH_SECTOR_MAGIC &&
 	    header1.magic == FLASH_SECTOR_MAGIC) {
@@ -109,7 +109,7 @@
 
 	flash_offset = flash_mem_context.base[flash_select] + offset
 						+ sizeof(struct flash_header);
-	spi_copy(data, flash_offset, size, "Flash Read");
+	spi_copy(data, flash_offset, size, NULL);
 
 	return 0;
 }
